sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -450,6 +450,19 @@ def run_pipeline(self,
         pipeline_options.DirectOptions).direct_runner_bundle_repeat
     self._num_workers = options.view_as(
         pipeline_options.DirectOptions).direct_num_workers or self._num_workers
+
+    # set direct workers running mode if it is defined with pipeline options.
+    running_mode = \
+      options.view_as(pipeline_options.DirectOptions).direct_running_mode
+    if running_mode == 'multi_threading':
+      self._default_environment = environments.EmbeddedPythonGrpcEnvironment()
+    elif running_mode == 'multi_processing':
+      command_bytes = b'%s -m apache_beam.runners.worker.sdk_worker_main' \
+                       % sys.executable.encode('ascii')
```

Comment:
Why not utf-8?

Reply:
I removed `command_bytes` because we don't need it anymore.